dataflow/src/org/checkerframework/dataflow/analysis/AnalysisResult.java

```diff
@@ -100,6 +100,11 @@ public void combine(AnalysisResult<A, S> other) {
         return treeLookup.get(tree);
     }
 
+    /** @return the corresponding {@link AssignmentNode} for a given {@link UnaryTree}. */
```

Comment:
When can this return null? Shouldn't all UnaryTrees be processed before this method is invoked? Therefore, wouldn't it be better to fail if a given tree is not in the map?